resources/puppetlabs/lein-ezbake/template/foss/ext/default.erb

```diff
@@ -11,12 +11,11 @@ JAVA_ARGS="<%= EZBake::Config[:java_args] %>"
 # These normally shouldn't need to be edited if using OS packages
 USER="<%= EZBake::Config[:user] %>"
 GROUP="<%= EZBake::Config[:group] %>"
-INSTALL_DIR="/opt/puppetlabs/server/apps/<%= EZBake::Config[:real_name] %>"
```

Comment:
ah, ok, thanks.  Yeah that might explain the failure I'm seeing locally.  I'll try changing that and see if it fixes my local tests... thanks